docs/intro/getting-started-with-mkdocs.rst

```diff
@@ -0,0 +1,73 @@
+Getting Started with MkDocs
+===========================
+
+MkDocs is a documentation generator that focuses on speed and simplicity.
+It has many great features including:
+
+* Preview your documentation as you write it
+* Easy customization with themes and extensions
+* Writing documentation with Markdown
+
+.. note::
+
+    MkDocs is a great choice for building technical documentation.
+    However, Read the Docs also supports :doc:`Sphinx </intro/getting-started-with-sphinx>`,
+    another tool for writing and building documentation.
+
+
+Quick start
+-----------
+
+Assuming you have Python already, `install MkDocs`_:
+
+.. sourcecode:: bash
+
+    $ pip install mkdocs
+
+Setup your MkDocs project:
+
+.. sourcecode:: bash
+
+    $ mkdocs new .
+
+This command creates ``mkdocs.yml`` which holds your MkDocs configuration,
+and ``docs/index.md`` which is the Markdown file
+that is the entry point for your documentation.
+
+You can edit this ``index.md`` file to add more details about your project
+and then you can build your documentation:
+
+.. sourcecode:: bash
+
+    $ mkdocs serve
+
+This command builds your Markdown files into HTML
+and starts a development server to browse your documentation.
+Open up http://127.0.0.1:8000/ in your web browser to see your documentation.
+You can make changes to your Markdown files and your docs will automatically rebuild.
+
+.. figure:: ../_static/images/first-steps/mkdocs-hello-world.png
```

Comment:
`_static` is the theme static file output directory, and it isn't necessary to put `/images/` under `/_static`. It can exist top level

Reply:
It doesn't **have** to be but it is easier if we do put it in `_static`. Putting images in `_static` gives us a deterministic place to link to with `:target:` and the `:target:` will be the same as the source.

Without it, putting the images in `_images` or somewhere else means the `:target:` would need to be set to the directory (not `_images`) where the images get copied to by Sphinx during generation which is not intuitive at all.